package.json

```diff
@@ -41,12 +41,11 @@
   "scripts": {
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
-    "test": "yarn-or-npm run test-node-all && yarn-or-npm run lint",
```

Comment:
Rather than totally removing `yarn-or-npm run lint` shouldn't we change it to `yarn-or-npm run lint:js`?

Reply:
I don't think so. We're kinda trying to be Ember-ish here, and in the Ember blueprint, `yarn test` runs `ember test` and `yarn lint:js` lints (and CI runs both) -- linting is separate from testing. That's what I was going for here -- make sense?